---
title: Publications
layout: default
---

<!-- Main -->
	<section class="wrapper style1">
		<div class="container">
			<div id="content">

				<!-- Content -->

					<article>
						<header>
							<h2>Publications</h2>
						</header>

<ul>
	<li><strong>2021</strong></li>
						<li>165. Sun X. J. Zhang, Q. Nie    Inferring latent temporal progression and regulatory networks form cross-sectional transcriptomic data of cancer samples, Accepted for publication, <strong><em>PLoS Computational Biology</em></strong>, 2021</li>
						<li>164.    Jin S, C. Guerrero-Juarez, L. Zhang, I. Chang, R. Ramos, C. Kuan, P. Myung, K. Plikus, Q. Nie.  Inference and analysis of cell-cell communication using CellChat. <strong><em>Nature Communications</em></strong>, 12(1088), 2021</li>
						<li>163.  Zhang L, Q. Nie. scMC learns biological variation through the alignment of multiple single cell genomics datasets. <strong><em>Genome Biology</em></strong>, 22:10, 2021</li>
						<li>162.   Zhao X, J Chen, P Xiao, J Feng, Q. Nie &amp; X Zhao. Identifying age-specific gene signatures of the human cerebral cortex with joint analysis of  transcriptomes and functional connections. <strong><em>Briefings in Bioinformatics</em></strong>, accepted for publication, 2021. <a href="https://doi.org/10.1093/bib/bbaa388">https://doi.org/10.1093/bib/bbaa388</a></li>
						<li>161.   Sha Y, S. Wang, F. Bocci, P. Zhou, Q. Nie. Inference of intercellular communications and multilayer gene-regulations of Epithelia-Mesenchymal transition from single-cell transcriptomic  data. <strong><em>Frontiers in Genetics</em></strong>, Jan. 8,  publication, 2021 https://doi.org/10.3389/fgene.2020.604585</li>
						<li>160.    Cang Z, Y. Wang, Q. Wang, K. Cho, W. Holmes, Q. Nie.  A multiscale model via single-cell  transcriptomics reveals robust patterning mechanism during early mammalian embryo development. <strong><em>PLoS Computational Biology</em></strong>, Accepted for publication, 2020.</li>
						<li><strong>2020</strong></li>
						<li>159. Fu L., L Zhang, E. Dollinger, Q. Peng, Q. Nie<sup>*</sup>, X. Xie<sup>*</sup>. Predicting transcription factor binding in single cells through deep learning. <strong><em>Science Advances</em></strong><strong>,</strong> 6:eaba9301, 2020. *:co-corresponding author.</li>
						<li>158.  Villarreal-Ponce A., M. Tiruneh, J. Lee, C Guerrero-Juarez, J. Kun, J. David, K. Dammeyer, R. Kell, J Kwong, P. Rabbani, Q. Nie, D. Ceradini.  Keratinocyte-Macrophage crosstalk by the  Nrf2/Ccl2/EGF signaling axis orchestrates tissue regeneration. <strong><em>Cell Reports</em></strong> 33:108417. 2020</li>
						<li>157. Lin Z, Suoqin Jin, Jefferson Chen<sup>1</sup>, Zhuorui Li, Zhongqi Lin, Li Tang, Qing Nie<sup>,*</sup>, Bogi Andersen<sup>*</sup> , Murine Interfollicular epidermal differentiation is gradualistic with GRHL3 controlling progression from stem to transition cell states. <strong><em>Nature Communications,</em></strong> 11:5434, 2020. <strong><em> </em></strong>*: co-corresponding author.</li>
						<li>156.  Dollinger E. D. Bergman, P. Zhou, S. Atwood, Q. Nie.  Divergent Resistance Mechanisms to Immunotherapy Explain Responses in Different Skin Cancers. <strong><em>Cancers</em></strong>, 12:2946, 2020.</li>
						<li>155.  Xie J , Chang Zhao , Jiamin Sun , Jiaxin Li , Fuzhang Yang , Jiao Wang , Qing Nie, Prediction of Essential Genes in Comparison States Using Machine Learning. <strong><em>IEEE/ACM Transactions on Computational Biology and Bioinformatics, </em></strong><em>Accepted for publication,2020</em></li>
						<li>154. Sha Y. S. S. Wang, P. Zhou, Q. Nie.  Inference and multiscale model of Epithelial-to-Mesenchymal transition via single-cell transcriptomic data. <strong><em>Nucleic Acids Research</em></strong>, 48(17), Pages 9505-9530. 2020</li>
						<li>153. Xiaole Sheng, Ziguang Lin, Cong Lv, Chunlei Shao, Xueyun Bi, Min Deng, Jiuzhi Xu, Christian F Guerrero-Juarez, Mengzhen Li, Xi Wu, Ran Zhao, Xu Yang, Guilin Li, Xiaowei Liu, Qingyu Wang, Qing Nie, Wei Cui, Shan Gao, Hongquan Zhang, Zhihua Liu, Yingzi Cong, Maksim V Plikus, Christopher J Lengner, Bogi Andersen, Fazheng Ren, Zhengquan Yu.  Cycling stem cells are radioresistant and regenerate the intestine.  <strong><em>Cell Reports</em></strong>,  32(4), Page 107952, 2020</li>
						<li>152.  Wang S. M Drummond, C Guerrero-Juarez, E Tarapore, AL. MacLean, Adam R. Stabell, S Wu, G Gutierrez, B That, C Benavente, Qing Nie<sup>*</sup>, S Atwood<sup>*</sup>. Single cell   transcriptomics of human epidermis identifies basal stem cell transition states.<strong><em>  Nature Communications</em></strong>,11:4239, 2020 *: co-corresponding author.</li>
						<li>151.  Zhu Y. Y Qiu, W Chen, Q Nie A Lander. Scaling a Dpp morphogen gradient through feedback control of receptors and co-receptors. <strong><em>Developmental Cell</em></strong>, 53,  724-739, 2020</li>
						<li>150.  Cang Z and Q. Nie. Inferring spatial and signaling relationships between cells from single cell transcriptomic data. <strong><em>Nature Communications</em></strong>, 11(2084), 2020.</li>
						<li>149.  Xie, J. F. Yang, J. Wang, M. Karikomi, Y. Yin, J. Sun, T. Wen, Q. Nie.  DNF: A differential network flow method to identify rewiring drivers for gene regulatory networks. <strong><em>Neurocomputing, </em></strong>to appear soon, 2020</li>
						<li>148. Song Y, Guerrero-Juarez, Zhongjian Chen, Yichen Tang, Xianghui Ma, Cong Lv,    Xueyun Bi, Min Deng<sup>1</sup>, Lina Bu, Yuhua Tian, Ruiqi Liu, Ran Zhao, Jiuzhi Xu, Xiaole Sheng, Sujuan Du, Yeqiang Liu, Yunlu Zhu, Shi-jun Shan, Hong-duo Chen, Yiqiang Zhao, Guangbiao Zhou, Jianwei Shuai, Fazheng Ren, Lixiang Xue, Zhaoxia Ying, Xing Dai, Christopher J. Lengner, Bogi Andersen, Maksim V. Plikus, Q. Nie, Zhengquan Yu. The Msi1-mTOR pathway drives the pathogenesis of mammary and extramammary Paget’s Disease.  <strong><em>Cell Research</em></strong>, 2020, https://doi.org/10.1038/s41422-020-0334-5</li>
						<li>147. Haensel, D<sup>#</sup>, S Jin<sup>#</sup>, P Sun, R Cinco, M Dragan, Q. Nguyen, Z, Xang, Y Gong, R Vu, A MacLean, K. Kessenbrock E. Gratton, Q. Nie*, X. Dai*.  Defining epidermal basal cell states during skin homeostasis and wound healing using single-cell transcriptomics. <strong><em>Cell Reports</em></strong>, 30, 3932-3947, 2020. *: co-corresponding author, <em><sup>#</sup></em>:equal-contributions</li>
						<li>146. Jin, S., L. Zhang, Q. Nie. scAI: an unsupervised approach for the integrative analysis of  parallel single-cell transcriptomic and epigenomic profiles. <strong><em>Genome Biology</em></strong>, 21(1):25, 2020</li>
						<li>145.  Greenberg E, M Marshall, S Jin, S Venkatesh, M Dragan, L Tsoi, J Gudjonsson, Q  Nie, Taskhashi, B Andersen. Circadian control of interferon-sensitive gene expression in murine skin. <strong><em>Proceedings of the National Academy of Sciences</em></strong>, 117(11) 5761-5771, 2020</li>
						<li>144. Kashgari G, L. Meinecke, W. Gordon, B. Ruiz, J. Yang, Y Xie, A. Ma, H. Ho, Q. Nie, K. Jester, B. Andersen.  Epithelial migration and non-adhesive periderm are required for digit separation during mammalian development. <strong><em>Developmental Cell</em></strong>,  52 (764), 1-15,  2020.</li>
						<ul>
						<li><strong>Cover Story </strong>of <em>Developmental Cell</em> (April 2020)</li>
						<li>“A sticky problem for digits” highlighted in<strong> <em>Science</em></strong>, 368(6487), p151, 2020</li>
						</ul>
						<li style="text-align: left;">143. Nie, Q, L. Qiao, Y. Qiu, L. Zhang, W. Zhao. Noise control and utility: from gene regulatory network to spatial patterning. <strong><em>Science China Mathematics</em></strong>, 63(3): 425-440. 2020</li>
						<li>142. A. Lander, Q. Nie, C. Sanchez-Tapia, A. Simonyan, F. Wan.  Regulatory feedback on receptor and non-receptor synthesis for robust signaling.  <strong><em>Developmental Dynamics</em></strong>, 249:383-409, 2020</li>
						<li>141. Xie, J, J. Sun, J. Feng, F. Yang, J. Wang, Tieqiao Wen &amp; Q. Nie. Kernel differential subgraph analysis to reveal the key period affecting glioblastoma. <strong><em>Biomolecules,</em></strong> 10:318. 2020</li>
						<li><strong>2019<br />
						</strong></li>
						<li>140. Zhang J, Q. Nie, T. Zhou. Revealing dynamic mechanisms of cell fate decisions from  single-cell transcriptomic data. <strong><em>Frontiers in Genetics</em></strong>, 10:1280.  2019</li>
						<li>139. Qiao L. Zhao W, Tang C, Nie, Q*, Zhang L*.  Network topologies that can achieve dual function of adaptation and noise attenuation. <strong><em>Cell Systems</em></strong>, Vol 9 (3), P271-285. E7, 2019. * co-corresponding authors.</li>
						<li>138.  Sun Y*, Jin S*, Lin X, Chen L, Qiao X, Jiang L, Zhou P. Johnston, Golshani P. Nie, Q. Holmes T, Nitz D, Xu X.  CA1-Projecting subiculum neurons facilitate object-place learning, <strong><em>Nature Neuroscience</em></strong>, 9/24, 2019. *: co-first authors</li>
						<li>137. Nie Q and Plikus M. Equal Opportunities in Stemness. <strong><em>Nature Cell Biology</em></strong>,  21, pages921–923, 2019</li>
						<li>136. Zheng X, Jin S, Nie Q*, Zou X*. scRCMF: Identification of cell subpopulations and transition states from single cell transcriptomes, <strong><em>IEEE Transactions on Biomedical Engineering, </em></strong>doi: 10.1109/TBME.2019.2937228, 2019.  * co-corresponding authors</li>
						<li>135. S. Wang, M. Karikomi, A. MacLean, Q. Nie. Cell Lineage and Communication Inference via Optimization for Single-cell Transcriptomics. Accepted for publication, <strong><em>Nucleic Acids Research</em></strong>, 2019</li>
						<li>134. C. Rackauckas and Q. Nie. Confederated Modular Differential Equation APIs for Accelerated Algorithm Development and Benchmark.&nbsp;<strong><em>Advances in Engineering Software</em></strong>, 132, 1-6, 2019</li>
						<li>133. Y. Wang, C. Guerrero-Juarez, Y. Qiu, H. Du, W. Chen, S. Figueroa, M. Plikus, Q. Nie. A Multi-scale Mathematical Model of Epidermal-dermal Interactions during Skin Wound Healing. Accepted for publication, <strong><em>Experimental Dermatology</em></strong>, 2019.</li>
						<li>132. Y. Qiu, W. Chen, and Q. Nie. A Hybrid Method for Stiff Reaction-diffusion Equations. Accepted for publication, <strong><em>Discrete and Continuous Dynamical Systems-B</em></strong>, 2019.</li>
						<li>131. C. Guerrero-Juarez, P. Dedhia, S. Jin, R. Ruiz-Vega, D. Ma, Y. Liu, K. Yamaga, O. Shestova, D. Gay, Z. Yang, K. Kessenbrock, Q. Nie, W. Pear, G. Cotsarelis, M. V. Plikus<sup>. </sup>Single-cell Analysis Reveals Fibroblast Heterogeneity and Myeloid-derived Adipocyte Progenitors in Skin Wounds, <strong><em>Nature Communications</em></strong>. 10(1):650, 2019.</li>
						<li>130. Y. Sha, D. Haensel, G. Guitierrez, H. Du, X. Dai, Q. Nie. Intermediate Cell States in Epithelial-to-Mesenchymal Transition. <strong><em>Physical Biology</em></strong>, 18:16(2):021001, 2019.</li>

					<li><strong>2018</strong><ul>
						<li>129. P. Sharma, A. MacLean, L. Meinecke, D. Clouthier, Q. Nie, T. Schilling. Transcriptomics Reveals Complex Kinetics of Dorsal-Ventral Patterning Gene Expression in the Mandibular Arch. <strong><em>The Journal of Genetics and Development</em></strong>, 2018:e23275, 2018.</li>
						<li>128. C. Rackauckas, T. Schilling, Q. Nie. Interdisciplinary Case Study:How Mathematicians and Biologists Found Order in Cellular Noise.&nbsp;<strong><em>iScience</em></strong> 8, 267-270, 2018. <strong>Featured as a cover article for iScience.</strong></li>
						<li>127. L. Meinecke, P. Sharma, H. Du, L. Zhang, Q. Nie*, T. Schilling*. Modeling Craniofacial Development Reveals Spatiotemporal Constrains on Robust Patterning of the Mandibular Arch. <strong><em>PLoS Computational Biology</em></strong> 14(11):e1006569. 2018. *co-corresponding authors.</li>
						<li>126. D. Haensel, P. Sun, A. MacLean, X. Ma, Y. Zhou, M. Stemmler, S. Brabletz, G. Berx, M Plolis, Q. Nie, T. Brabletz, X. Dai. An Ovol2-Zeb1 Transcriptional Circuit Regulates&nbsp;&nbsp; Epithelial Directional Migration and Proliferation. <strong><em>EMBO Reports</em></strong>, e46273. 2018.</li>
						<li>125. Y. Qiu, W. Chen, Q. Nie. Stochastic Dynamics of Cell Lineages in Tissue<br>
						Homeostasis. <strong><em>Discrete and Continuous Dynamic Systems–B</em></strong>,<br>
						doi:&nbsp;<a href="http://dx.doi.org/10.3934/dcdsb.2018339">10.3934/dcdsb.2018339</a>, 2018.</li>
						<li>124. Q. Nie. Stem Cells: a Window of Opportunity of Low-Dimensional EMT Space.&nbsp;<strong><em>Oncotarget</em></strong>, Vol. 9 (61), pp31790, 2018</li>
						<li>123. C. Li, L. Zhang, Q. Nie. Landscape reveals critical network structures for sharpening gene expression boundaries. BMC Systems Biology.12:67, 2018.</li>
						<li>122. C. Rackaukas, T. Schilling, Q. Nie. Mean-Independent Noise Control of Cell Fates via Intermediate State. <em>iScience</em>, DOI:&nbsp;<a href="https://doi.org/10.1016/j.isci.2018.04.002">https://doi.org/10.1016/j.isci.2018.04.002</a>, Vol. 3, Pages 11-20. 2018.</li>
						<li>121. Y. He, Q. Zuo, J. Edwards, K. Zhao, J. Leo, W. Cai. Q. Nie, B. L. J. Song. DNA Methylationand Regulatory Element during Chicken Germline Stem Cell Differentiation. Stem Cell Report, 10:6, pp1793-1806, 2018.</li>
						<li>120. A. MacLean, H. Tian, Q. Nie. Exploring intermediate cell states through the lens of single cells. <em>Current Opinion in Systems Biology</em>. Vol 9. Pages 32-41, 2018.</li>
						<li>119. S. Jin, A. MacLean, T. Peng, and Q. Nie.&nbsp;&nbsp; scEpath: Energy landscape-based inference of transition probabilities and cellular trajectories from single-cell transcriptomic data, <em>Bioinformatics</em>, 34:12, Pages 2077-2086, 2018.</li>
						<li>118. J. Lei, Q. Nie<sup>*</sup>, D. Chen*. A single-cell epigenetic model for parental psychological stress-induced transgenerational reprogramming in offspring. <em>Biology of Reproduction</em>. 98(6):846-855.. *: co-corresponding author, 2018.</li>
						<li>117. H. Du, Y, Wang, D. Haensel, B. Lee, X. Dai, Q. Nie. Multiscale modeling of layer formation of epidermis. <em>PLoS Computational. Biology</em>, 4(2):e1006006. <a href="https://doi.org/10.1371/journal.pcbi.1006006">https://doi.org/10.1371/journal.pcbi.1006006</a>, 2018.</li>
						<li>116. P. Yu, Q. Nie<sup>*</sup>, C. Tang*, L. Zhang*. Nanog induced intermediate state in regulating stem cell differentiation and reprogramming. *: co-corresponding author <em>BMC Systems Biology, 12:22 </em>doi.org/10.1186/s12918-018-0552-3, 2018.</li>

					<li><strong>2017</strong><ul>
						<li>115. J. Xie, D. Lu, J. Li, J. Wang, Y. Li, Q. Nie. Kernel differential subgraph reveals dynamic changes in biomolecular network. J<em>. Bioinformatics and Computational&nbsp;</em><em>Biology</em>, 4:1750027. doi:10.1142/S0219720017500275. 2017.<br>
						<li>114. Y. Guo, Q. Nie, A. MacLean, Y. Li, J. Lei, S. Li. Multiscale modeling of inflammation-induced tumorigenesis reveals competing oncogenic and onco-protective roles for inflammation, <em>Cancer Research</em>. doi:10.1158/0008-5472.CAN-17-1662, 2017.<br>
						<li>113. Qixuan, Wang, Ji Won Oh, Anukriti Dhar, Jonathan Le, Shelby C. Jocoy, Antoni R. Rossi, Hoang T. Ha, Melisa A. Fuentes, Manda P. Nguyen, Julien Legrand, Eve Kandyba, Jung Chul Kim, &nbsp;&nbsp;&nbsp;Moonkyu Kim, Krzysztof Kobielak, Kiarash Khosrotehrani, Qing Nie*, Maksim V. Plikus*. A multi-scale model for the hair follicle reveals a pacemaker mechanism driving rapid hair growth patterning, *co-corresponding author, <em>eLife 2017;</em>6:e22772, 2017.<br>
						<li>112. C. Rackauckas, Q. Nie. DifferentialEquations.jl-A Performant and Feature-Rich Ecosystem for Solving Differential equations in Julia, <em>Journal of Open Research Software</em>, 5(1), 2017.<br>
						<li>111. T. Peng, L. Liu, A. MacLean, C. Wong, W. Zhao, and Q. Nie. A Mathematical model of mechanotransduction reveals how mechanical memory regulates mesenchymal stem cell fate decisions. <em>BMC Systems Biology</em>, 11:55, 2017. PMCID: PMC5434622.<br>
						<li>110. C. Li<sup>**</sup>, T. Hong<sup>**</sup>, Y. Tung, Y. Yen, H. Hsu<sup>2</sup>, Y. Lu, M. Chang, Q. Nie<sup>3,*</sup>, J. Chen<sup>*</sup>. MicroRNA Filters Hox Temporal Transcription Noise to Confer Boundary Formation in the Spinal Cord<strong>&nbsp;&nbsp;</strong>**: equal-contribution, *:co-corresponding authors. <em>Nature Communication</em>, 8:14685, 2017.<br>
						<li>109. A. Li, S. Figueroa, T. Jiang, P. Wu, R. Widelitz, Q. Nie, C. Chuong. Diverse feather shape evolution enabled by coupling anisotropic signaling modules with self-organizing branching programme. <em>Nature Communication</em>, 8:14139, 2017, PMID: 28106042<br>
						<li>108. W. R. Holmes, Nabora Soledad Reyes de Mochel, Qixian Wang, Huijing Du, Michael Chiang, Olivier Cinquin, Ken W.Y. Cho, Qing Nie, Gene expression noise enhances robust organization of the early mammalian balstocyst., 13(1): e1005320,<br>
						<a href="http://dx.doi.org/10.1371/journal.pcbi.1005320">http://dx.doi.org/10.1371/journal.pcbi.1005320</a>, <em>PLoS Computational Biology</em>, 2017. PMID 28114387<br>
						<li>107. Q. Wang, W. R. Holmes, J. Julian, T. Schilling, Q. Nie. Cell sorting and noise-induced cell plasticity coordinate to sharpen boundaries between gene expression domains, 13(1):<br>
						e1005307, <em>PLoS Computational Biology</em>, 2017, PMID: 28135279<br>
						<li>106. C. Rackauacks, Q. Nie, Adaptive Methods for Stochastic Differential Equations via Natural Embeddings and Rejection Sampling with Memory, Vol 22(7), p2731-2761, <em>Discrete and Continuous Dynamic systems</em> – B, 2016<br>
						<li>105. J. Xie, J. Xu, C. Nie, Q. Nie. Machine Learning of Swimming Data via Wisdom of Crowd and Regression Analysis. <em>Mathematical Biosciences and Engineering</em>. 14(2), 511-527, doi:10.3934/mbe.2017031, 2017.
			   	<li><strong>2016</strong><ul>
					 	<li>104. J. Lo, L. Zheng, Q. Nie.   A Hybrid Continuous-Discrete Method for Stochastic Reaction-Diffusion Processes. <em>Royal Society Open Science</em>, 3:160485. 2016.</li>
					 	<li>103. C. Li, T. Hong, C. Webb, H. Karner, S. Sun, Q. Nie. A Self-Enhanced Transport Mechanism through Long Non-Coding RNAs for X Chromosome Inactivation. 6-31517,doi:10.1038/srep31517 <em>Scientific Report</em>, 2016.</li>
						<li>102. W. Chen. Q. Nie, T. Yi, and C. Chou, Modeling of Yeast Mating Reveals Robustness Strategies for Cell-Cell Interactions. Accepted for publication, <em>PLoS Computational Biology</em>, 2016</li>
						<li>101. Chunhe Li, Hong TIan, and Qing Nie. Quantifying the landscape and kinetic paths for epithelial-mesenchymal transition from a core circuit. DOI: 10.1039/c6cp03174a, Physical Chemistry Chemical Physics, 2016</li>
						<li>100. Jiajun Zhang, Qing Nie, and Tianshou Zhou, A moment-convergence method for stochastic analysis of biochemical reaction networks. <em>The Journal of Chemical Physics</em>, 144, 194109 (2016); <a href="http://dx.doi.org/10.1063/1.4950767">http://dx.doi.org/10.1063/1.4950767</a>, 2016</li>
						<li>99. Catherine Ta, Qing Nie<sup>*</sup>, Hong Tian<sup>*</sup>. Controlling stochasticity in epithelial-mesenchymal transition through multiple intermediate cellular states, Accepted for publication, <em>Discrete and Continuous Dynamical Systems-B</em>, 2016. Co-corresponding author</li>
						<li>98. Jinzhi. Lei, Wing-Cheong Lo, and Qing Nie Mathematical models of morphogen dynamics and growth control. Accepted for publication, <em>Annals of Mathematical Sciences and Applications</em>. 2016</li>
						<li>97. J. Julian, L. Zheng, C. Rackaukas, M. Digman, E. Gratton. Q. Nie, T. Schilling. Noise modulation in retinoic acid signaling sharpens segmental boundaries of gene expression in the zebrafish hindbrain<em>, eLife, </em>e14034, <em>2016</li>
						</em>96<em>. </em><em> Xu, N. Olivas, T. Ikrar, T. Peng, T. C Holmes, Q. Nie, and Y. Shi, </em>Primary Visual Cortex Shows Laminar Specific and Balanced Circuit Organization of Excitatory and Inhibitory Synaptic Connectivity<em>. J. of Physiology, </em> Vol 594 (7), pp 1891, DOI: 10.1113/JP271891, 2016</li>
						</ul></li><li><strong>2015</strong><ul>
						<li>95. A. Cinquin, L. Zheng, P. Taytor, L. Zhang, M. Chiang, J. Snow, Q. Nie, and O. Cinquin. Semi-permeable diffusion barriers enhance patterning robustness in C. elegans germ line. Developmental Cell, 35, pp405-417, 2015.</li>
						<li>94. T. Hong, K. Watanabe, C. Ta, A. Villarreal-Ponce, Q. Nie* and X. Dai*. An Ovol2-Zeb1 Mutual Inhibitory Circuit Governs Bidirectional and Multi-step Transition between Epithelial and Mesenchymal States. *co-corresponding author, PLoS Computational Biology, 11(11): e1004569. doi:10.1371/journal.pcbi.1004569, 2015.</li>
						<li>93. H. Du, Q. Nie*, W. Holmes*. The Interplay between Wnt Mediated Expansion and Negative Regulation of Growth Promotes Robust Intestinal Crypt Structure and Homeostasis. *co-corresponding author, PLoS Computational Biology, 11(8): e1004285. doi:10.1371/journal.pcbi.1004285 , 2015.</li>
						<li>92. J. Xie, C. Xiang, J. Ma, J. Tan, T. Wen, J. Lei, and Q. Nie. An Adaptive Hybrid Algorithm for Global Network Alignmen. IEEE/ACM Transactions on Computational Biology and Bioinformatics. doi:10.1109/TCBB.2015.2465957, 2015.</li>
						<li>91. A. Li, Y. Lai, S. Figueroa. T. Yan. R. Widelitz, K. Kobielak, Q. Nie, and C. Chuong. Deciphering Principles of Morphogenesis from Temporal and Spatial Patterns on the Integument. Developmental Dynamics, doi: 10.1002/dvdy.24281, 2015.</li>
						<li>90. C. Ta, D. Wang, and Q. Nie. An Integration Factor Method for Stochastic and Stiff Reaction-diffusion Systems. J. of Computational Physics, 295:505-522, 2015.</li>
						<li>89. T. Hong, E. Fung, L. Zhang, G. Huynh, E. Monuki, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja89.pdf">Semi-adaptive Response and Noise Attenuation in BMP Signaling.</a> Journal of the Royal Society Interface, 12(107) 2015.</li>
						<li>88. J. Xie, Z. Zhou, J. Ma, C. Xiang. Q. Nie, and W. Zhang. Graphics Processing Unit-based Alignment of Protein Interaction Networks. IET Systems Biology, IET Systems Biology, 10.1049/iet-syb.2014.0052 2015.</li>
						<li>87. D. Wang, W. Chen, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja87.pdf">Semi-implicit Integration Factor Methods on Sparse Grids for High-dimensional Systems.</a> Journal of Computational Physics, 292, 43-55,2015</li>
						<li>86. W. Lo, S. Zhou, F. Wan, A. Lander, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja86.pdf">Robust and Precise Morphogen-Mediated Patterning: Tradeoffs, Constraints, and Mechanisms.</a> Journal of the Royal Society Interface, 12, 2015.</li>
						<li>85. C. Chou, T. Moore, Q. Nie, and T. Yi. <a href="http://www.math.uci.edu/~qnie/Publications/ja85.pdf">Alternative Cell Polarity Behaviors Arise from Changes in G-Protein Spatial Dynamics.</a> IET Systems Biology, 9(2), pp52-63, 2015.</li>
						</ul></li><li><strong>2014</strong><ul>
						<li>84. A. Gord, W. Holmes, X. Dai, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja84.pdf">Computational Modeling of Epidermal Stratification Highlights the Importance of Asymmetric Cell Division for Predictable and Robust Layer Formation.</a> Journal of the Royal Society Interface, 11:99, 2014.</li>
						<li>83. B. Lee, A. Villarreal-Ponce, M. Fallahi, J. Ovadia, P. Sun. Q. Yu, S. Ito, S. Sinha, Q. Nie, and X. Dai. <a href="http://www.math.uci.edu/~qnie/Publications/ja83.pdf">Transcriptional Mechanism Link Epithelial Plasticity to Adhesion and Differentiation of Epidermal Progenitor Cells.</a> Developmental Cell, 29(1), p47-58, 2014. &#8211;</li>
						Highlighted in <a href="http://www.math.uci.edu/~qnie/Publications/ja83_preview.pdf">preview article</a>, 29(1), pp1-2, Developmental Cell.<br />
						<li>82. J. Lei, S. Levin and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja82.pdf">A Mathematical Model of Adult Stem Cells Regeneration with Crosstalk between Genetic and Epigenetic Regulation.</a> Proceeding of National Academy of Sciences, USA, E880-E887, doi: 10.1073/pnas.1324267111, 2014. &#8211; Highlighted in PNAS Early Edition; commentary article: PNAS 111 (10) pp. 3653-3654.</li>
						<li>81. W. Holmes and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja81.pdf">Interactions and Tradeoffs between Cell Recruitment, Proliferation, and Differentiation Affect CNS Regeneration.</a> 106:1528-1536, 2014.</li>
						<li>80. Z. Zheng, S. Christley, W. Chiu, I. Blitz, X. Xie, K. Cho, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja80.pdf">Inference of the Xenopus tropicalis Embryonic Regulatory Network and Spatial Gene Expression Patterns.</a> BMC Systems Biology, 8:3, doi:101186/1752-0509-8-3, 2014.</li>
						<li>79. D. Wang, L. Zhang, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja79.pdf">Array-representation Integration Factor Method for High-dimensional Systems.</a> Journal of Computational Physics, 258, pp585-600, 2014.</li>
						<li>78. G. Ye, M. Tang, J. Cai, Q. Nie, X. Xie. <a href="http://www.math.uci.edu/~qnie/Publications/ja78.pdf">Low-Rank Regularization for Learning Gene Expression Programs.</a> PLoS One, 9(1), doi:10.1371, 2014.</li>
						</ul></li><li><strong>2013</strong><ul>
						<li>77. X. Liu, S. Johnson, S. Liu, D. KanoJia, W. Yue, U. Singn,Qian Wang, Qi, Wang, Q. Nie and H. Chen. <a href="http://www.math.uci.edu/~qnie/Publications/ja77.pdf">Non-linear Growth Kinetics of Breast Cancer Stem Cells: Implications for Cancer Stem Cell Targeted Therapy.</a> 3:2473, DOI: 10.1038/srep02473, Scientific Reports, 2013.</li>
						<li>76. M. Chen, L. Wang, C. Liu, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja76.pdf">Noise Attenuation in the ON and OFF States of Biological Switches.</a> <a href="http://pubs.acs.org/doi/abs/10.1021/sb400044g">Featured and cover article</a>, ACS Synthetic Biology, 2, pp587-593, 2013, DOI: 10.1021/sb400044g.</li>
						<li>75. J. Ovadia and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja75.pdf">Numerical Methods for Two-Dimensional Stem Cell Tissue Growth.</a> Journal of Scientific Computing.2013, DOI 10.1007/s10915-013-9728-6.</li>
						<li>74. A. Li, M. Chen, T. Jiang, P. Wu, Q. Nie, R. Widelitz, C. Choung. <a href="http://www.math.uci.edu/~qnie/Publications/ja74.pdf">Shaping Organs by a Wnt/Notch/non-muscle Myosin Module Which Orients Feather Bud Elongation. </a>Proceeding of the National Academy of Sciences, USA, E1452-E1461, 2013.</li>
						<li>73. J. Zhang, Q. Nie, M. He, and T. Zhou. <a href="http://www.math.uci.edu/~qnie/Publications/ja73.pdf">An Effective Method for Computing the Noise in Biochemical Networks.</a> Journal of Chemical Physics, 138, 084106, p. 1-16, 2013.</li>
						<li>72. J. Ovadia and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja72.pdf">Stem Cell Niche As an Inherent Cause of Undulating Epithelial Morphologies.</a> Biophysical Journal, 104 (1): 237-46, 2013.</li>
						<li>71. J. Lei, D. Wang, Y. Song, Q. Nie and Y. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/ja71.pdf">Robustness of Morphogen Gradients with “Bucket Brigade” Transport through membrane-associated Non-receptor.</a> Discrete and Continuous Dynamical Systems-B, 18(3), 2013</li>
						</ul></li><li><strong>2012</strong><ul>
						<li>70. T. Schilling, Q. Nie, A. Lander. <a href="http://www.math.uci.edu/~qnie/Publications/ja70.pdf">Dynamics and Precision in Retinoic Acid Morphogen Gradients.</a> Current Opinion in Genetics &amp; Development, 22 (6), 2012.</li>
						<li>69. C. Chou, T. Moore, S. Chang, Q. Nie and T. Yi. <a href="http://www.math.uci.edu/~qnie/Publications/ja69.pdf">Signaling Regulated Endocytosis and Exocytosis Lead to Mating Pheromone Concentration Dependent Morphologies in Yeast. </a>FEBS Letters, 586 (23), Page 4208-4214, 2012.</li>
						<li>68. L. Zheng, M. Chen, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja68.pdf">External Noise Control in Inherently Stochastic Biological Systems.</a> Journal of Mathematical Physics, 53, 115616, 201.</li>
						<li>67. L. Zhang, K. Radtke, L. Zheng, T. Schilling, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja67.pdf">Noise Drives Sharpening of Gene Experssion Boundaries in Zebrafish Hindbrain. Nature Molecular Systems Biology</a>. Nature Molecular Systems Biology, 8:613, 2012.</li>
						<li>66. L. Zhang, A. Lander, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja66.pdf">A Reaction-Diffusion Mechanism Influences Cell Lineage Progression as a Basis for Formation, Regeneration, and Stability of Intestinal Crypts.</a> BMC Systems Biology, 6:93, 2012.</li>
						<li>65. Q. Nie, <a href="http://www.math.uci.edu/~qnie/Publications/ja65.pdf">Challenges for Training at the Interface</a>. Bioengineering &amp; Biomedical Science, 2:3, 1000e105, 2012.</li>
						<li>64. C. Chan, X. Liu, L.Wang, L. Bardwell, Q. Nie*, and G. Enciso*. <a href="http://www.math.uci.edu/~qnie/Publications/ja64.pdf">Protein Scaffolds Can Enhance the Bistability of Multisite Phosphorylation Systems.</a> *Co-corresponding authors, PLoS Computational Biology, 8(6) e1002551, 2012.</li>
						<li>63. A. Cai, K. Radtke, A. Linville, A. Lander, Q. Nie*, T. Schilling. <a href="http://www.math.uci.edu/~qnie/Publications/ja63.pdf">*Cellular Retinoic Acid-Binding Proteins Are Essential For Hindbrainpatterning and Signal Robustness in Zebrafish.</a> *Co-corresponding authors, Development, 139, 2150-2155. 2012.</li>
						<li>62. W. Lo, L. Chen, M. Wang, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja62.pdf">A Robust and Efficient Method for Steady State Patterns in Reaction-Diffusion Systems.</a> J. of Computational Physics, 231, 5062-5077, 2012.</li>
						<li>61. S. Zhou, W. Lo, J. Suhalim, M. Digman, E. Gratton, Q. Nie, and A. Lander. <a href="http://www.math.uci.edu/~qnie/Publications/ja61.pdf">Free Extracellular Diffusion Creates the Dpp Morphogen Gradient of the Drosophila Wing Disc. </a>Current Biology, 22, 668-675, 2012.</li>
						</ul></li><li><strong>2011</strong><ul>
						<li>60. C. Chou, L. Bardwell, Q. Nie*, T. Yi*. <a href="http://www.math.uci.edu/~qnie/Publications/ja60.pdf">Noise Filtering Tradeoffs in Spatial Gradient Sensing and Cell Polarization Response.</a> *Co-corresponding authors, BMC Systems Biology, 5:196 2011.</li>
						<li>59. S. Zhao, J. Ovadia, X. Liu, Y.T. Zhang. Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja59.pdf">Operator Splitting Implicit Integration Factor Methods for Stiff Reaction-diffusion-Advection Systems.</a> J. of Computational Physics, 230(15), pp 5996-6009, 2011.</li>
						<li>58. Z. Zheng, C. Chou, T. M. Yi, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja58.pdf">Mathematical Analysis of Steady-State Solutions in Compartment and Continuum models of Cell Polarization.</a> Mathematical Biosciences and Engineering. 8(4), 2011.</li>
						<li>57. J. Lei, F.Y.M. Wan, A. Lander, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja57.pdf">Robustness of Signaling Gradient in Drosophila Wing Imaginal Disc.</a> Discrete and Continuous Dynamical Systems-B, 16(3), 2011.</li>
						</ul></li><li><strong>2010</strong><ul>
						<li>56. C. Chou, Lo, W, Gokoffski, K., Zhang, Y., Wan, F., Calof, A., and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja56.pdf">Spatial Dynamics of Multi-stage Cell Lineages in Tissue Stratification.</a> Biophysical Journal, 99(10), 2010.</li>
						<li>55. L. Wang, Q. Nie, G. Enciso. <a href="http://www.math.uci.edu/~qnie/Publications/ja55.pdf">Non-essential Sites Improve Phosphorylation Switch.</a> Biophysical Journal, 99(6), 2010.</li>
						<li>54. S. Haney, L. Bardwell, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja54.pdf">Ultrasensitive Responses and Specificity in Cell Signaling.</a> BMC Systems Biology, 4 (119), 2010</li>
						<li>53. S. Christley, B. Lee, X. Dai and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja53.pdf">Integrative multicellular biological modeling: a case study of 3D epidermal development using GPU algorithms.</a> BMC Systems Biology, 4(107), 2010</li>
						<li>52. L. Wang, J. Xin, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja52.pdf">A Critical Quantity for Noise Attenuation in Feedback Systems.</a> PLoS Computational Biology, 6(4): e1000764, 2010.</li>
						<li>51. X. Liu and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja51.pdf">A Compact Integration Factor Methods for Complex Domains and Adaptive Mesh Refinement.</a> J. of Computational Physics. 229, pp 5692-5706, 2010.</li>
						<li>50. A.D. Lander, Q. Nie, B. Vargas, and F.Y.M. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/ja50.pdf">Size-normalized Robustness of Dpp Gradient in Drosophila Wing Imaginal Disc.</a> J. Mat. Mech. Struc. (JoMMS). Accepted 2010.</li>
						<li>49. X. Liu, L. Bardwell, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja49.pdf">A Combination of Multisite Phosphorylation and Substrate Sequestration Produces Switch-Like Responses.</a> Biophysicsl Journal. 98(8), pp1396-1407, 2010</li>
						</ul></li><li><strong>2009</strong><ul>
						<li>48. J. Lei, G. He, H. Liu, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja48.pdf">A Delay Model for Noise-Induced Bi-Directional Switching</a>. Nonlinearity, 22, pp 2845-2859, 2009.</li>
						<li>47. S. Christley, Q. Nie and X. Xie. I<a href="http://www.math.uci.edu/~qnie/Publications/ja47.pdf">ncorporating Existing Network Information into Gene Network Inference</a>.  PLoS ONE, 4(8): e6799, 2009.</li>
						<li>46. J. Wells, B. Lee, A. Cai, A. Karapetyan, W. Lee, E. Rugg, S. Sinha, Q. Nie, X. Dai. <a href="http://www.math.uci.edu/~qnie/Publications/ja46.pdf">Ovol2 Suppresses Cell Cycling and Terminal Differentiation of Keratinocytes by Directly Repressing C-Myc And Notch1</a>.  J. of Biological Chemistry, 284, pp 29125-29135, 2009.</li>
						<li>45. A. Cai, Y. Peng, J.. Wells, X. Dai, Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja45.pdf">Multi-scale Modeling for Threshold Dependent Differentiation.</a> Math. Model of Nat. Phenom. 4(4), pp 103-117. 2009</li>
						<li>44. X. Li and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja44.pdf">A High-Order Boundary Integral Method for Surface Diffusions on Elastically Stressed Axi-Symmetric Rods.</a> J. of Computational Physics, 228(12), pp 4625-4637, 2009.</li>
						<li>43. A. Lander, W. Lo, Q. Nie, and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/ja43.pdf">The Measure of Success: Contraints, Objectives, and Tradeoffs in Morphogen-Mediated Patterning.</a> Cold Spring Harb Perspectives in Biology, 1:a002022, 2009.</li>
						<li>42. A. Lander, K. Gokoffski, F. Wan, Q. Nie, and A. Calof. <a href="http://www.math.uci.edu/~qnie/Publications/ja42.pdf">Cell Lineages and the Logic of Proliferative Control.</a> PLoS Biology, 7(1):e1000015, 2009.</li>
						<li>41. W. Lo, C. Chou, K. Gokoffski, F. Wan, A. Lander, A. Calof and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja41.pdf">Feedback Regulation in Multistage Cell Lineages.</a> Mathematical Biosciences and Engineering, 6(1), pp 59-82, 2009.</li>
						<li>40. Y. Zhou, J. He and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja40.pdf">A Comparative Runtime Analysis of Heuristic Algorithms for Satisfiability Problems.</a> Artificial Intelligence, 173(2):240-257, 2009.</li>
						<li>39. A. Lander, Q. Nie, F. Wan, Y. Zhang. <a href="http://www.math.uci.edu/~qnie/Publications/ja39.pdf">Localized Ectopic Expression of Dpp Receptors in a Drosophila Embryo.</a> Studies in Applied Mathematics, 123, pp 175-214, 2009.</li>
						</ul></li><li><strong>2008</strong><ul>
						<li>38.T. Moore, C.S. Chou, Q. Nie, Jeon, N.L. and T. M. Yi. <a href="http://www.math.uci.edu/~qnie/Publications/ja38.pdf">Robust Spatial Sensing of Mating Pheromone Gradients by Yeast Cells.</a> PLoS ONE, 3(12):e3865, 2008.</li>
						<li>37. S. Chou, S. Zhao, Y. Song, H. LIu, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja37.pdf">Fus3-Triggered Tec1 Degradation Modulates Mating Transcriptional Output during the Pheromone Response.</a> Nature Molecular Systems Biology, 4:212, 2008. [<a href="http://www.math.uci.edu/~qnie/Publications/ja37_sup.pdf">supplementary Info</a>]</li>
						<li>36. C. Chou, Q. Nie, and T. Yi. <a href="http://www.math.uci.edu/~qnie/Publications/ja36.pdf">Modeling Robustness Tradeoffs in Yeast Cell Polarization Induced by Spatial Gradients</a>. PLoS ONE, 3(9):e3103, 2008.</li>
						<li>35. Q. Nie, F.Y.M. Wan, Y-T Zhang, and X-F Liu. <a href="http://www.math.uci.edu/~qnie/Publications/ja35.pdf">Compact Integration Factor Methods in High Spatial Dimensions.</a> Journal of Computational Physics, 227(10), pp 5238-5255, 2008.</li>
						<li>34. D. Iron, A. Syed, H. Theisen, T. Lukacsovich, M. Naghibi, L.J. Marsh, F.Y.M. Wan, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja34.pdf">The Role of Feedback in the Formation of Morphogen Territories.</a> Mathematical Biosciences of Engineering, 5(2), pp 277-298, 2008.</li>
						</ul></li><li><strong>2007</strong><ul>
						<li>33. R. White, Q. Nie, L. Lander, and T. Schilling. <a href="http://www.math.uci.edu/~qnie/Publications/ja33.pdf">Complex Regulation of cyp26a1 Creates a Robust Retinoic Acid Gradient in the Zebrafish Embryo.</a> PLoS Biology, 5(11), e304, 2007. (<a href="http://www.math.uci.edu/~qnie/Publications/ja33_press.pdf">Press Release</a>)</li>
						<li>32. Y. Zhang, A. Lander and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja32.pdf">Computational Analysis of BMP Gradients in Dorsal-ventral Patterning of the Zebrafish Embryo.</a> Journal of Theoretical Biology, 248, pp 579-589, 2007.</li>
						<li>31. L. Bardwell, X. Zou, Q. Nie, and N. Kamorova. <a href="http://www.math.uci.edu/~qnie/Publications/ja31.pdf">Mathematical Models of Specificity in Cell Signaling.</a> Biophysical Journal. 92, pp 3425-3441, 2007.</li>
						<li>30. T. Yi, S. Chen, C. Chou, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja30.pdf">Modeling Yeast Cell Polarization Induced by Pheromone Gradients.</a> J. of Statistical Physics. 128(1), pp 193-207, 2007.</li>
						<li>29. C. Chou, Y. Zhang, R. Zhao, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/ja29.pdf">Numerical Methods for Stiff Reaction-Diffusion Systems.</a> Discrete and Continuous Dynamical Systems-B, 7(3), pp 515-525, 2007.</li>
						<li>28. X. Li, V. Cristini, Q. Nie, and J. Lowengrub.  <a href="http://www.math.uci.edu/~qnie/Publications/ja28.pdf">Nonlinear Three-Dimensional Simulation of Solid Tumor Growth.</a> Discrete and Continuous Dynamical System-B, 7(3), pp 581-604, 2007.</li>
						<li>27. A. Lander, Q. Nie and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/ja27.pdf">Membrane Associated Non-receptors and Morphogen Gradients.</a> Bulletin of Mathematical Biology, 69, pp 33-54, 2007.</li>
						<li>26. H. Theisen, A. Syed, B. Nguyen, T. Lukasovich, J. Purcell, G. Srivastava, D. Irons, K. Gaudenz, Q. Nie, F. Wan, M. Waterman, and J. Marsh. <a href="http://www.math.uci.edu/~qnie/Publications/ja26.pdf">Wingless Directly Represses DPP Morphogen Expression via an Armadillo/TCF/Brinker Complex.</a> PLoS ONE, 2(1): e142, 2007.</li>
						<li>25. X. Li and Q. Nie. Surface Diffusion on Stressed Solid Surface. Communications in Computational Physics, 2(1), pp 73-86, 2007.</li>
						</ul></li><li><strong>2000-2006</strong><ul>
						<li>24. Q. Nie, Y. Zhang and R. Zhao. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pdf.24.pdf">Efficient Semi-implicit Schemes for Stiff Systems. </a>Journal of Computational Physics, 214, pp 521-537, 2006.</li>
						<li>23. A. Lander, Q. Nie and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pdf.23.pdf">Internalization and End Flux in Morphogen Gradient Formation.</a> Journal of Computational and Applied Mathematics, 190(1-2), pp 232-251, 2006.</li>
						<li>22. N. Komarova, X. Zou, Q. Nie and L. Bardwell. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub22.pdf">A Theoretical Framework for Specificity in Cell Signaling.</a> Nature Molecular Systems Biology, 1:2005.0023, 2005. Supplement</li>
						<li>21. C. Mizutant, Q. Nie, F. Wan, Y. Zhang, P. Vilmos, E. Bier, L. Marsh and A. Lander. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub21.pdf">Formation of the BMP Activity Gradient in the Drosophia Embryo.</a> Developmental Cell, 8(6), pp 915-924, 2005. Supplement</li>
						<li>20. Y. Lou, Q. Nie and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub20.pdf">Effects of Sog on Dpp-Receptor Binding.</a> SIAM J. on Applied Math., 66 (5), pp 1748-1771, 2005.</li>
						<li>19. A. Lander, Q. Nie and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub19.pdf">Spatially Distributed Morphogen Production and Morphogen Gradient Formation.</a>  Mathematical Biosciences and Engineering, 2(2), pp 239-262, 2005.</li>
						<li>18. A. Lander, Q. Nie, B. Vargas and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub18.pdf">Aggregation of a Distributed Source in Morphogee Gradient Formation.</a> Studies in Applied Mathematics, 114(4), pp 343-374, 2005.</li>
						<li>17. X. Li, K. Thornton, Q. Nie, P. Voorhees and J. Lowengrub. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub17.pdf">Two- and Three-dimensional Equilibrium Morphology of a Misfitting Particle and the Gibbs-Thomson Effect.</a> Acta Materialia, Vol 52/20, pp 5829-5843, 2004.</li>
						<li>16. Y. Lou, Q. Nie, and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub16.pdf">Nonlinear Eigenvalue Problems in the Stability Analysis of Morphogen Gradients.</a>  Studies in Applied Mathematics, Vol 113, pp 183-215, 2004.</li>
						<li>15. X. Li, J. Lowengrub, Q. Nie, V. Cristini and P. Leo. Microstructural Evolution in Three-Dimensional Inhomogeneous Elastic Media. Metall. Mater. Tran. A, 34A(7), pp 1421-1431, 2003.</li>
						<li>14. V. Cristini, J. Lowengrub and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pdf.14.pdf">Nonlinear Simulation of Tumor Growth.</a> J. of Mathematical Biology, 46(3), pp 191-224, 2003.</li>
						<li>13. A. Lander, Q. Nie and F. Wan. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub13.pdf">Do Morphogen Gradients Arise by Diffusion? </a>Developmental Cell, Vol. 2, no. 6, pp 785-796, 2002. Previews</li>
						<li>12. Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub12.pdf">The Nonlinear Evolution of Vortex Sheets with Surface Tension in Axi-symmetric Flows.</a> J. of Computational Physics, 174, pp 438-459, 2001.</li>
						<li>11. Q. Nie and F. Tian. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub11.pdf">Singularities in Hele-Shaw Flows Driven by a Multipole.</a> SIAM J. on Applied Mathematics, 62(2), pp 385-406, 2001.</li>
						<li>10. P. Leo, J. Lowengrub and Q. Nie. On an Elastically Induced Splitting Instability. Acta Mater. 49, pp. 2761-2772, 2001.</li>
						<li>9. P. Leo, J. Lowengrub and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub9.pdf">Microstructural Evolution in Inhomogeneous and Anisotropic Elastic Media.</a> J. of Computational Physics, 157, pp 44-88, 2000.</li>
						</ul></li><li><strong>1995-1999</strong><ul>
						<li>8. P. Constantin, Q. Nie and N. Schorghofer. Front Formation in an Active Scalar. Physical Review E, 60(3), pp. 2858-2863, 1999.</li>
						<li>7. P. Constantin, Q. Nie and S. Tanveer. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub7.pdf">Bounds for Second Order Structure Functions and Energy Spectrum in Turbulence.</a> Physics of Fluids, 11(8), pp. 2251-2256, 1999.</li>
						<li>6. Q. Nie and S. Tanveer. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub6.pdf">A Note on Third Order Structure Functions in Turbulence.</a> Proc. Royal Soc. London A, 455, pp 1615-1636, 1999.</li>
						<li>5. P. Constantin, Q. Nie and N. Schorghofer. Nonsingular Surface Quasi-Geostrophic flows. Physics Letters A 241, pp 168-172, 1998.</li>
						<li>4. Q. Nie and G. Baker. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/pub4.pdf">Application of Adaptive Quadrature to Axi-symmetric Vortex Sheet Motion.</a> J. of Computational Physics 143, pp. 49-69, 1998.</li>
						<li>3. G. Baker and Q. Nie. The Asymptotic Motion of an Accelerating, Thick Layer of Inviscid Liquid. Physics of Fluids 10(1), pp. 101-112, 1998.</li>
						<li>2. Q. Nie and F. Tian. Singularities in Hele-Shaw Flows. SIAM J. on Applied Mathematics 58(1), pp. 34-54, 1998.</li>
						<li>1. Q. Nie and S. Tanveer. The Stability of a Two-dimensional Rising Bubble. Physics of Fluids 7 (6), pp. 1292-1306, 1995.</li>
						</ul></li><li><strong>Published Refereed Proceeding Articles and Book Chapters, Editor for Special Issues</strong><ul>
						<li>10. Jiang Xie, Junfu Xu, Celine Nie, and Qing Nie. “Prediction on Performance of Age Group Swimming Using Machine Learning&#8221;, The Third International Conference of High Performance Computing and Applications, Lecture Notes in Computer Science (LNCS), Springer, Switzerland, 2016.</li>
						<li>9 . Xiaoying Han and Qing Nie, Editor, Special Issue on “Analysis and quantification of noise effects in biological systems”. Discrete and Continuous Dynamical Systems -B, 2016.</li>
						<li>8. Schilling TF, Sosnik J and Nie Q). Visualizing retinoic acid morphogen gradients. Methods in Cell Biology 133, 139-163. In The Zebrafish: Cellular and Molecular Biology, Part A, Cellular Biology (eds. HW Detrich III, M Westerfield, LI Zon). Elsevier, Academic Press. 2016</li>
						<li>7. Qing Nie. Systems Biology. The Princeton Companion to Applied Mathematics, editors: N. Higham, M. Dennis, P. Glendinning, P. Martin, F. Santosa. Princeton University Press, Princeton, NJ. 2014.6.</li>
						<li>6. Youfang Cao, Claire Liang, Hammad Naveed, Yingzi Li, Meng Chen and Qing Nie, Modeling spatial population dynamics of stem cell lineage in tissue growth, Proc. 34th Annual International Conference of the IEEE EMBS San Diego, California USA, 5502-5505, 2012</li>
						<li>5. Y. Cao, C. Liang, H. Naveed, Y. Li, M. Chen, and Q. Nie. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/proceeding5.pdf">Modeling Spatial Population Dynamics of Stem Cell Lineage in Tissue Growth.</a> Proceeding 34th Annual International Conference of the IEEE EMBS, San Diego, California, 5502-5505, 2012.</li>
						<li>4. Q. Nie and Y.-T. Zhang. Cell Biology Modeling Development, Encyclopedia of Applied and Computational Mathematics, Springer, accepted, 2011.</li>
						<li>3. X. F. Liu and Q. Nie. Spatially-localized scaffold proteins may facilitate to transmit long-range signals. Acta Mathematicia, Scientia, 29B (6), pp 1657-1669, 2009</li>
						<li>2. J. Kao, Q. Nie, A. Teng, F. Wan, A. Lander, and J. Marsh. Can Morphogen Activity be Enhanced by its Inhibitors? Proceedings of the 2nd MIT Conference on Computational Fluid and Solid Mechanics, pp1729-1733, 2003.</li>
						<li>1. Q. Nie, S. Tanveer, T. Dupont and X. Li. <a href="http://www.math.uci.edu/~qnie/Publications/pdf/proceeding1.pdf">Singularity Formation in Free-Surface Stokes Flows.</a> Contemporary Mathematics, Vol 306, pp 147-165, 2002.</li>
					</ul></li></ul>
					</article>

			</div>
		</div>
	</section>
