---
title: Qing Nie
layout: default
---

<!-- Main -->
	<section class="wrapper style1">
		<div class="container">
			<div id="content">

			<!-- Content -->
			  <article>
			
	<h3>Line Meinecke<h3b>, Postdoctoral Scholar<br>
      <script language="JavaScript">
                            var username="lina.meinecke";
                            var hostname="uci.edu";
                            var linktext = username + "@" + hostname;
                            document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a>");
			  </script>

			  </h3b></h3>
	<div style="max-width: 50em; text-align: justify; font-size:0.9em; ">
	  <span class="image left"><img src="{{ site.baseurl }}/assets/images/meinecke1.jpg" style="width:10em" alt="" /></span>  

	  &nbsp;
	  <p style="padding-bottom: 4em">
I joined the Nie Lab in September 2016 as a postdoctoral research scholar. Before that I completed my PhD in scientific computing at the Department of Information Technology at Uppsala University.
	  </p>
	  
	  <h3>Research Interests</h3>

	  <p>
	  <b>Stochastic reaction diffusion models in single cells</b><br>
Living cells are highly organized in space, e.g. the nucleus contains the DNA in eukaryotic cells and important reaction complexes are often bound to the membrane. In order to simulate single cells accurately we therefore need to take their spatial organization and the molecules’ diffusive movement into account. 
	  In addition, important molecules inside cells are often only present at very low copy numbers, e.g. there is only one DNA molecule. Hence, stochastic models are necessary to understand many biological phenomena.</p>
	  <p>
	  In my research, I have focused on developing accurate and efficient computational tools to simulate diffusion as a discrete stochastic jump process. I have in particular focused on an unstructured domain discretization in order to represent complicated geometries and on macromolecular crowding effects.
	  </p>

<b>Pattern formation in embryonic development</b><br>
<p>Recently I am interested in extending these spatial stochastic methods to multicellular systems to investigate the formation of gene expression patterns during embryonic development and how robustness these patterns are in the presence of noise.
	  </p>

	  <h3>Background</h3>
<ul><li>2016	PhD in scientific computing with specialization in numerical analysis, Division of Scientific Computing, Department of Information Technology, Uppsala University.<br>
  Thesis title: Stochastic Simulation of Multiscale Reaction-Diffusion Models via First Exit Times</li>
  <li>2011	Diploma (MSc) in Mathematics, Technical University of Munich.</li></ul>


  <h3>Teaching/Outreach</h3>
<p>Finite Element Methods (TA)<br>
Scientific Computing 2 (Responsible teacher, TA)<br>
Higher Mathematics for Engineers (TA)<br>
President of the SIAM student chapter at Uppsala University
</p>
	  
	


<h3>Publications</h3>
<ul><li>
<b>Meinecke, L.</b> Multiscale modeling of diffusion in a crowded environment. Submitted. Preprint at: arXiv:1603.05605.
  </li>
  <li>
  <b>Meinecke, L.</b>, Eriksson, M. Excluded volume effects in on- and off-lattice reaction-diffusion models. Accepted for publication in IET Systems Biology. Preprint at: arXiv:1604.06660.  
  </li>
  <li>
<b>Meinecke, L.</b>, Engblom, S., Hellander, A., and Lötstedt, P. Analysis and design of jump coefficients in discrete stochastic diffusion models. 
SIAM J. Sci. Comput. 38(1), A55-A83, 2016.  
</li>
<li>
Lötstedt, P., and <b>Meinecke, L.</b> Simulation of stochastic diffusion via first exit times. 
J. Comput. Phys. 300, 862-886, 2015.
</li>
<li>
Meinecke, L., and Lötstedt, P. Stochastic diffusion processes on Cartesian meshes. 
J. Comput. Appl. Math. 294, 1-11, 2016.  
</li>
</ul>


<h3>Presentations</h3>
<ul><li>
  07/2016:	Multiscale modeling of diffusion in the crowded cell environment.
  European Conference on Mathematical and Theoretical Biology, Nottingham.
  </li>
<li>	
01/2016:	Multiscale modeling of diffusion in the crowded cell environment. (Poster) 
Stochastic Dynamical Systems in Biology: Numerical Methods and Applications, Isaac Newton Institute, Cambridge.
  </li>
  <li>
08/2015:	A multiscale approach to diffusion simulation in the crowded cell environment.
		BIT Circus 2015, Umeå
  </li>
  <li>
08/2015:	Analysis and design of jump coefficients in discrete stochastic diffusion models.
		ICIAM15, Beijing
  </li>
  <li>
06/2015:	Multiscale diffusion simulations in biology.
		Mathematical Methods in Systems Biology, Dublin
  </li>
  <li>
01/2015:	Analysis and design of jump coefficients in discrete stochastic diffusion models.
		Seminar, UC Santa Barbara, 2015,
  </li>
  <li>
01/2015:	Stochastic diffusion simulation. (Poster)
		5th Annual Southern California Systems Biology Conference, Irvine
  </li>
  <li>
07/2014:	Stochastic Diffusion Processes in Systems Biology.
		SIAM Annual Meeting 2014, Chicago
  </li>
  <li>
06/2014:	Stochastic Simulation of Diffusion on Unstructured Meshes via First Exit Times. 
European Conference on Mathematical and Theoretical Biology, Gothenburg
  </li>
  <li>
09/2013:	Stochastic Simulation of Diffusion on Unstructured Meshes via First Exit Times. 
Summer School on Numerical Methods for Stochastic Differential Equations, Vienna
  </li>
  <li>
08/2013:	Stochastic Simulation of Diffusion on Unstructured Meshes via First Exit Times. 
		ENUMATH 2013, Lausanne
  </li>
  <li>
12/2012:	Diffusion Simulation on Unstructured Meshes via First Exit Times. (Poster)
		4th Swedish Meeting on Mathematics in Biology, Lund
  </li>
  <li>
08/2012:	Stochastic Simulation of Reaction-Diffusion Processes.
		BIT Circus 2012, Copenhagen
  </li>
  <li>
07/2012:	Diffusion Simulation on Unstructured Meshes via First Exit Times. (Poster)
Workshop on Stochastic Modeling of Reaction-Diffusion Processes in Biology, Oxford
</li>


	  </article>

	</div>
	</div>
</section>
