---
title: Qing Nie
layout: default
---

<!-- Main -->
	<section class="wrapper style1">
		<div class="container">
			<div id="content">

			<!-- Content -->
			  <article>
			
			  <h3>Tao Peng<h3b>, PhD Student<br>
      <script language="JavaScript">
                            var username="taop";
                            var hostname="uci.edu";
                            var linktext = username + "@" + hostname;
                            document.write("<a href='" + "mail" + "to:" + username + "@" + hostname + "'>" + linktext + "</a>");
      </script>

			  </h3b></h3>
	<div style="max-width: 50em; min-height: 30em; text-align: justify; font-size:0.9em; ">
	  <span class="image left"><img src="{{ site.baseurl }}/assets/images/tpeng1.jpg" style="width:10em" alt="" /></span>  
      
<p>
      I joined the Nie lab in June 2013 after I completed the Mathematical, Computational, and Systems Biology (MCSB) Gateway Program at UC Irvine. I previous studied at Wuhan University where I earned a B.S. and a M.S. in computational mathematics with a B.S.B.A. in business administration. My research interests are computational biology, bioinformatics, and numerical analysis.
       </p>

       <p>
       Go to my <a href="https://www.researchgate.net/profile/Tao_Peng42">ResearchGate</a> profile. 
       </p>

      <br>
      <p>
      My research focuses on data-driven methods to understand mechanisms of complex biological phenomena such as the fate decision of stem cells, drug screening on cancers, and so on. Many biological data have emerged recently from sequencing data at the molecular level to the live imaging data at the tissue level with new technological developments. Extracting, cleaning, integrating, and fitting the different types of data from multiple levels are still challenging. I have built mathematical models and developed computational and bioinformatics tools to study the core mechanisms of the biological systems using dynamical theory of continuous and discrete models, network analysis, statistical inference, and machine learning methods in my previous publications and current projects.  For example, to explore the underlying mechanisms of the drug resistance of p38 MAPK in treating Multiple Myeloma (MM), we developed a novel systems biology approach by integrating liquid chromatography-mass spectrometry and reverse phase protein array data from human MM cell lines with computational pathway models in which the unknown parameters were inferred using a proposed novel algorithm called Modularized Factor Graph. </p>
      <p>
      Measurement of gene expression levels for multiple genes in single cells provides a powerful approach to study heterogeneity of cell populations and cellular plasticity. While the expression levels of multiple genes in each cell are available in such data, the potential connections among the cells (e.g. the lineage relationship) are not directly evident from the measurement. Classifying cellular states and identifying transitions among those states are challenging due to many factors, including the small number of cells versus the large number of genes collected in the data. As a result, we aim to develop a machine learning method to construct the 2D transition map of different types of cells based on the single cell data, such as scRNA-seq and qPCR. </p>

<br>


<h3>Publications</h3>

<p>
10. A mathematical model of mechanotransduction reveals How mechanical nemory regulates mesenchymal stem cell fate decisions <br>
<b>Tao Peng</b>*, Linan Liu*, Adam L MacLean, Chi Wut Wong, Weian Zhao, Qing Nie (*contributed equally to this work) <br>
Submitted
</p>


<p>
9. A multi-scale model for the hair follicle reveals heterogeneous skin domains driving rapid spatiotemporal hair growth patterning <br>
Qixuan Wang, Ji Won Oh, Hye-Lim Lee, Anukriti Dhar, <b>Tao Peng</b>, Raul Ramos, Christian Fernando Guerrero-Juarez, Xiaojie Wang, Jonathan Le, Melisa A. Fuentes, Shelby C. Jocoy, Antoni R. Rossi, Brian Vu, Kim Pham, Xiaoyang Wang, Nanda Maya Mali, Jung Min Park, Hyunsu Lee, Julien Legrand, Eve Kandyba, Jung Chul Kim, Moonkyu Kim, John Foley, Zhengquan Yu, Krzysztof Kobielak, Bogi Andersen, Kiarash Khosrotehrani, Qing Nie, Maksim V. Plikus <br>
Submitted
</p>

<p>
8. Gene expression noise enhances robust organization of the early mammalian blastocyst <br>
William R. Holmes, Nabora Soledad Reyes de Mochel, QixuanWang, Huijing Du, <b>Tao Peng</b>, Michael Chiang, Olivier Cinquin, Ken W.Y. Cho and Qing Nie <br>
Plos Computational Biology, accepted.
</p>


<p>
7. Primary visual cortex shows laminar-specific and balanced circuit organization of excitatory and inhibitory synaptic connectivity.<br>
Xiangmin Xu, Nicholas D. Olivas, Taruna Ikrar, <b>Tao Peng</b>, Todd C. Holmes, Qing Nie, and Yulin Shi. <br>
The Journal of Physiology, 2016.
</p>

<p> 
6. Characterization of p38 MAPK isoforms for drug resistance study using systems biology approach.<br>
Huiming Peng*, <b>Tao Peng</b>*, Jianguo Wen*, David A. Engler, RisK. Matsunami, Jing Su, Le Zhang, Chung-Che Jeff Chang, and Xiaobo Zhou. (*contributed equally to this work)<br>
Bioinformatics, 2014
</p>


<p> 
5. Modeling cell-cell interactions in regulating multiple myeloma initiating cell fate <br>
<b>Peng, Tao</b>*, Huiming Peng*, Dong Soon Choi*, Jing Su, Chung-Che Chang, and Xiaobo Zhou. (*contributed equally to this work) <br>
IEEE Journal of Biomedical and Health Informatics, 2014.
</p>



<p>
4. Systematically studying kinase inhibitor induced signaling network signatures by integrating both therapeutic and side effects <br>
Hongwei Shao, <b>Tao Peng</b>, Zhiwei Ji, Jing Su, and Xiaobo Zhou. <br>
PloS one, 2013
</p>

<p>
3. Cytokine combination therapy prediction for bone remodeling in tissue engineering based on the intracellular signaling pathway <br>
Sun, Xiaoqiang, Jing Su, Jiguang Bao, <b> Tao Peng</b>, Le Zhang, Yuanyuan Zhang, Yunzhi Yang, and Xiaobo Zhou.<br> 
Biomaterials, 2012.
</p>

<p>
2. Understanding inhibition of viral proteins on type I IFN signaling pathways with modeling and optimization <br>
Zou, Xiufen, Xueshuang Xiang, Yan Chen, <b>Tao Peng</b>, Xuelian Luo, and Zishu Pan. <br> 
Journal of theoretical biology, 2010.
</p>

<p>
1. Modeling specificity in the yeast MAPK signaling networks <br>
Zou, Xiufen, <b>Tao Peng</b>, and Zishu Pan. <br> 
Journal of theoretical biology, 2008.
</p>


<br>
<h3>Conference Poster Presentations </h3>
<ul>

<li>
4. Hongwei Shao, Chenglin Liu, Dongmin Guo, Caty Chung, Tao Peng, Stephan Schrer, Jing Su, and Xiaobo Zhou, ”itNETZ: the Signature-based Drug Discovery Platform”, LINCS Data Forum 2013, Boston MA (2013) </li>

<li>
3. Jing Su, Chenglin Liu, Caty Chung, Tao Peng, Hande Kk, Christopher Mader, Amar Koleti, Stephan Schrer, and Xiaobo Zhou, ”pLINDAW: A Fuzzy Query Based Data Warehouse System for Real-time Pan LINCS Data Integration and Visualization”, The NIH LINCS Fall Consortium Meeting, Washington DC (2012) </li>

<li>
2. Tao Peng, Dong Soon Choi, Huiming Peng, Jing Su, Hongyan Wang, Youli Zu, Chung-Che (Jeff) Chang, and Xiaobo Zhou, ”Systematically Model Cell- Cell Interactions in Regulating Multiple Myeloma Initiating Cell Fate”, The NIH LINCS Fall Consortium Meeting, Washington DC (2012) </li>

<li>
1. Jing Su, Tao Peng, Jianguo Wen, Chung-Che (Jeff) Chang, Youli Zu, and Xiaobo Zhou, ”Integrating Imaging, Genomics, and Functional Proteomics Analysis for Cellular Signature Discovery”, The NIH LINCS Fall Consortium Meeting, Washington DC (2011) </li>
</ul>
</div>

<!--?php PubmedList('MacLean+A[Author]'); ?-->

			 
      </article>
    </div>
  </div>
</section>
